package.json

```diff
@@ -61,6 +61,7 @@
     "@turf/area": "^6.0.1",
     "@turf/bbox": "^6.0.1",
     "@turf/centroid": "^6.0.2",
+    "acorn": "^7.1.1",
```

Comment:
Is it necessary that this lives in `dependencies`, rather than `devDependencies`?

Reply:
Good catch!

Reply:
Moved in https://github.com/plotly/plotly.js/pull/4800/commits/8a96b50696ff6fe286f5e5d626d2de4164addc88.